src/components/Scorecards/ScorecardDetailsPage/ScorecardsTableCard/RuleResultDetails.tsx

```diff
@@ -44,10 +48,19 @@ export const RuleResultDetails = ({
   hideWeight,
 }: RuleResultDetailsProps) => {
   const classes = useStyles();
+
+  const [open, setOpen] = useState(false);
+  const showExpandButton = rule.rule.description || rule.rule.title;
+  const hasTitle = !!rule.rule.title;
   const isFailing = rule.score === 0;
-  
+
   return (
     <ListItem alignItems="flex-start">
+      {showExpandButton && (
```

Comment:
played around with this for a bit but any way we can get these to be centered?
![image](https://user-images.githubusercontent.com/2001609/155665782-4ff451c6-6eb0-4c5c-a21a-03ff501f2c79.png)
maybe even abandoning the `List` component altogether and just using the raw `Grid` (even outside in `ScorecardResultDetails`? if it's taking too long no worries, just a small nitpick